container-registry/vulnerability-notification-function/.mvn/wrapper/MavenWrapperDownloader.java

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
We try to stay away from this stuff.

Reply:
The maven wrapper shouldn't be in our samples.

Reply:
removed